Anlab.Core/Domain/MailMessage.cs

```diff
@@ -28,5 +28,9 @@ public MailMessage()
         public DateTime? SentAt { get; set; }
 
         public DateTime CreatedAt { get; set; }
+
+        public Order Order { get; set; }
```

Comment:
@jSylvestre sure we could, just don't see how any of this is worth the time/effort.  just trying to send an email here